email/sender.js

```diff
@@ -7,10 +7,12 @@ module.exports = {
     let account = await getConfig(mailDetails.fromEmailAddress);
     for (var i = 0; i < account.length; i++) {
       if (mailDetails.fromEmailAddress == account[i].email) {
-        var mailKey = "Sent to : " + mailDetails.toEmailAddress + " " + mailDetails.subject.slice(0, 10); // needs change : THINK
+        var mailKey = "Sent to : " + mailDetails.toEmailAddress + " cc: " + mailDetails.ccEmailAddress + " bcc: " + mailDetails.bccEmailAddress + " " + mailDetails.subject.slice(0, 10); // needs change : THINK
```

Comment:
This was not supposed to be edited . please revert

Reply:
#26